src/components/Header.tsx

```diff
@@ -0,0 +1,10 @@
+import Image from "next/image";
+import DesktopLogo from "../public/images/desktop_logo.svg";
+
+export default function Header() {
+  return (
+    <header className="bg-black">
+      <Image src={DesktopLogo} alt={"로고"} width={75} height={35} />
```

Comment:
```suggestion
      <Image src="/images/desktop_logo.svg" alt={"로고"} width={75} height={35} />
```
보통 next.js에선 이렇게 쓰긴 해용 이거 정해야 할 것 같아요